packages/agent-utils-fs/package.json

```diff
@@ -10,6 +10,7 @@
     "@evo-ninja/agent-utils": "~0.1.0"
   },
   "devDependencies": {
+    "spawn-command": "0.0.2-1",
```

Comment:
is this intended?

Reply:
no this looks like an incorrect merge, as it's listed below on line 15